src/__tests__/cardholder-name.ts

```diff
@@ -0,0 +1,76 @@
+import { cardholderName } from "../cardholder-name";
+import type { Verification } from "../types";
+
+describe("cardholderName", () => {
+  describe.each([
+    [
+      "returns false for non-string types",
+      [
+        [0, { isValid: false, isPotentiallyValid: false }],
+        [0, { isValid: false, isPotentiallyValid: false }],
+        [123, { isValid: false, isPotentiallyValid: false }],
+        [1234, { isValid: false, isPotentiallyValid: false }],
+        [12345, { isValid: false, isPotentiallyValid: false }],
+        [557016, { isValid: false, isPotentiallyValid: false }],
+        [-1234, { isValid: false, isPotentiallyValid: false }],
+        [-10, { isValid: false, isPotentiallyValid: false }],
+        [0 / 0, { isValid: false, isPotentiallyValid: false }],
+        [Infinity, { isValid: false, isPotentiallyValid: false }],
+        [null, { isValid: false, isPotentiallyValid: false }],
+        [[], { isValid: false, isPotentiallyValid: false }],
+        [{}, { isValid: false, isPotentiallyValid: false }],
+      ],
+    ],
+
+    [
+      "returns false strings that are longer than 255 characters",
+      [
+        [
+          "this name is 256 chracters aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
+          { isValid: false, isPotentiallyValid: false },
+        ],
+      ],
+    ],
+
+    [
+      "accepts valid cardholder names",
+      [
+        ["name", { isValid: true, isPotentiallyValid: true }],
+        ["first last", { isValid: true, isPotentiallyValid: true }],
+        [
+          "this name is 255 chracters aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa",
+          { isValid: true, isPotentiallyValid: true },
+        ],
+        [
+          "first last another name",
```

Comment:
wondering if we change these examples to be globally inclusive

Reply:
```suggestion
          "name with many spaces",
```